CogniteSdk.Types/Annotations/AnnotationData.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+using System.Collections.Generic;
+
+namespace CogniteSdk
+{
+
+    /// <summary>
+    /// A bounding volume represents a region in a point cloud
+    /// </summary>
+    public class BoundingVolume: ObjectMetadataMixin
+    {
+        /// <summary>
+        /// The region of the annotation defined by a list of geometry primitives (cylinder and box).
+        /// </summary>
+        public List<Geometry> Region {get; set;}
+        /// <summary>
+        /// The asset this annotation is pointing to
+        /// </summary>
+        public AssetRef AssetRef{get; set;}
+    }
+    /// <summary>
+    /// A 3D geometry model represented by exactly one of `cylinder` and `box`.
+    /// </summary>
+    public class Geometry
+    {
+        /// <summary>
+        /// A box in 3D space, defined by a 4x4 row-major homogeneous transformation matrix that rotates and
+        /// translates a unit box centered at the origin to it's location and orientation in 3D space.
+        /// </summary>
+        public Box Box {get; set;}
+        /// <summary>
+        /// A cylinder in 3D space, defined by the centers of two sides and the radius.
+        /// </summary>
+        public Cylinder Cylinder {get; set;}
+    }
+    /// <summary>
+    /// A box in 3D space, defined by a 4x4 row-major homogeneous transformation matrix that rotates and
+    /// translates a unit box centered at the origin to it's location and orientation in 3D space.
+    /// </summary>
+    public class Box: ObjectMetadataMixin {
+        /// <summary>
+        /// The homogeneous transformation matrix
+        /// </summary>
+        public float[] Matrix {get; set;}
+    }
+    /// <summary>
+    /// A cylinder in 3D space, defined by the centers of two sides and the radius.
+    /// </summary>
+    public class Cylinder: ObjectMetadataMixin {
+        /// <summary>
+        /// The center of the first cap.
+        /// </summary>
+        public float[] CenterA {get; set;}
+        /// <summary>
+        /// The center of the second cap.
+        /// </summary>
+        public float[] CenterB {get; set;}
+        /// <summary>
+        /// The radius of the cylinder.
+        /// </summary>  
+        public float Radius {get; set;}
+
```

Comment:
Remove newline

Reply:
fixed in [8a6f4ca](https://github.com/cognitedata/cognite-sdk-dotnet/pull/287/commits/8a6f4ca65ad825bfa2d79a3230cc4ddc834da8e1)